ai-ml/gke-ray/rayserve/stable-diffusion/ray-cluster.yaml

```diff
@@ -27,7 +27,7 @@ spec:
       spec:
         containers:
         - name: ray-head
-          image: rayproject/ray-ml:2.9.0
+          image: rayproject/ray-ml:2.36.1
```

Comment:
We're you able to test the RayServe example too? I don't think this tag exists:
```
docker pull rayproject/ray-ml:2.36.1
Error response from daemon: manifest for rayproject/ray-ml:2.36.1 not found: manifest unknown: manifest unknown
```

Reply:
Consider using 2.36.1.999f76-py311 or 2.36.1.999f76-py310

Reply:
my bad, updated